doc/source/sdk/v2/migrating.md

```diff
@@ -0,0 +1,124 @@
+2. Run `npm install`
+
+#### Running locally
+In order to run your app within Zendesk you will need the [Zendesk Apps Tools (ZAT)](https://github.com/zendesk/zendesk_apps_tools). Once you've installed ZAT you will need two separate terminal tabs, one to watch and build this project and another serve it using ZAT:
+
+- `webpack --watch`
+- `zat server --path=./dist`
+
+#### Migrating your templates
+Copy your existing app's templates into src/templates.
+
+e.g.
+- `cd iframe_app_template`
+- `cp ../legacy_app/templates/* src/templates/`
+
+#### Migrating your app.css
+Copy your app.css into src/stylesheets with a new name and import it into app.scss.
+
+e.g.
+- `cd iframe_app_template`
+- `cp ../legacy_app/app.css src/stylesheets/common.scss`
+- `echo "@import './common.scss';" >> src/stylesheets/app.scss`
+
+#### Migrating your app.js
+Copy your app.js to src/javascripts with a new name and update its structure.
+
+e.g.
+- `cd iframe_app_template`
+- `cp ../legacy_app/app.js src/javascripts/legacy_app.js`
+
+##### Updating the structure
+Because your app is now compiled by webpack you will need to update its structure. Rather than returning the app object from an anonymous function you will need to export it using ES2015 syntax.
+
+e.g.
+Before:
+```js
+(function() {
+
+  return {
+    events: {
+      'app.created': function() {
+        this.switchTo('main');
+      }
+    }
+  };
+
+}());
+```
+
+After:
+```js
+import BaseApp from './base_app'; // ZAF v1 shims
+
+var App = {
+  events: {
+    'app.created': function() {
+      this.switchTo('main');
+    }
+  }
+};
+
+export default BaseApp.extend(App);
+```
+
+#### Copying your manifest.json
+Copy your manifest.json to dist/manifest.json and update the locations object.
+
+e.g.
+- `cd iframe_app_template`
+- `cp ../legacy_app/manifest.json dist/manifest.json`
+
+##### Updating locations
+In ZAF v2 your locations object needs to define url paths that point to HTML files. When migrating you can point all your locations to the same HTML.
+
+e.g.
+Before:
+```json
+{
+  "location": ["ticket_sidebar", "new_ticket_sidebar"]
+}
+```
+
+After:
+```json
+{
+  "location": {
+    "zendesk": {
+      "ticket_sidebar":     "assets/iframe.html",
+      "new_ticket_sidebar": "assets/iframe.html",
+    }
+  }
+}
+```
+
+#### Now what?
+If you're incredibly lucky your app may just work after following these steps. However, more likely you will need to update your code to use the new ZAF SDK v2 APIs. See [ZAF SDK v2](./zaf_v2#getting-started) for some examples on how the new APIs work. Additionally, you may need to rewrite some functionality, which may be missing from the Iframe App Template or simply not present in ZAF v2. In any case, this guide should give you a good starting point to get your app ready for the future. If you get stuck please don't hesitate to ask for help. You can reach us at support@zendesk.com.
```

Comment:
Original:
> If you're incredibly lucky your app may just work after following these steps. However, more likely you will need to update your code to use the new ZAF SDK v2 APIs...

The tone is a little off. Almost sounds like we're disparaging our solution. Suggested edit:
> Even after following these steps, your app may not work unless you update your code to use the new ZAF SDK v2 APIs...